bin/ember-template-lint.js

```diff
@@ -354,10 +370,15 @@ async function run() {
   if (options.printPending) {
     return printPending(results, options);
   } else {
-    if (results.errorCount || results.warningCount || (options.includeTodo && results.todoCount)) {
+    let hasErrors = results.errorCount > 0;
+    let hasWarnings = results.warningCount > 0;
+    let hasTodos = options.includeTodo && results.todoCount;
+    let hasUpdatedTodos = options.updateTodo && (todoInfo.added || todoInfo.removed);
```

Comment:
I think this needs to be updated for `--fix` support. Because we don't want to allow `--fix --update-todo` together. 

More directly: `todoInfo.removed` may have values **regardless** of `options.updateTodo`.

Reply:
I wonder if we can do:

```suggestion
    let hasUpdatedTodos = todoInfo.added > 0 || todoInfo.removed > 0;
```

We only expect `added` / `removed` to be set when `--update-todo` is set, so we can leverage that and simplify the boolean condition maybe... 🤔

Reply:
Again, we don't want to compare to 0 - we want 0 states to be output, but yes, we can remove the `options.updateTodo` portion.

Reply:
I'm actually waffling a bit on whether we should have output with `--fix`. I think it makes sense to me to report output with an explicit action like `--update-todo`. With `--fix` I feel like this is a side-effect of the fixing, in that we're just being good citizens with cleaning up ghost files. I'd be inclined to leave it out. WDYT?

Reply:
>  since we only set todoInfo to an object in that condition

I think we ought to initialize it regardless (makes it simpler logic wise, since we don't have to guard all over the place).

> I'm actually waffling a bit on whether we should have output with --fix. I think it makes sense to me to report output with an explicit action like --update-todo. With --fix I feel like this is a side-effect of the fixing, in that we're just being good citizens with cleaning up ghost files. I'd be inclined to leave it out. WDYT?

We should mirror what we do for `--fix`ed rules (non-todo), do we tell folks that we fixed/mutated a bunch of things? I _think_ a `--fix` run is completely silent (it emits the same output as `ember-template-lint .`, the remaining issues).

tldr; we can leave `--fix` info out for now 😸